common/lib/api-client/models.js

```diff
@@ -319,10 +323,6 @@ module.exports = {
         jsonApi: 'hasMany',
         type: 'moves',
       },
-      events: {
```

Comment:
Are there likely to be any effects from removing this property from the allocations model?

Reply:
Nope

Reply:
To clarify, the allocations BE does not return events for an allocation since it does not have any. 

This is an artifact of when `events` in a previous iteration was used as a catch-all generic way to update resource. Cancelling an allocation used to be done by posting a cancel event to the `/allocations/:id/event` endpoint. We now post to `/allocations/:id/cancel`.